### PR TITLE
Feat 87 -- list box stimulus

### PR DESCRIPTION
**Pull request by Giri-anandh**
This PR contains the fix which generates error when trying to access disabled items from a list box and testcases are also written to verify the functionality.

### CODE REVIEW

File: Source/Test/Tests/ListBoxTests/Stimulus/LB_NameArray_Tests.vi
Comment (kavin1730) [inline Source/Test/Tests/ListBoxTests/Stimulus/LB_NameArray_Tests.vi:1]: - [ ] Remove the breakpoint
image

**Review by kavin1730 — CHANGES_REQUESTED** [review]
Update the case structure directly from search array in all these APIs.

- [x] Source/APIs/Stimulus/ControlValue/Controls/Listbox/SetValueByArrayIndex.vi
- [x] Source/APIs/Stimulus/ControlValue/Controls/Listbox/SetValueByArrayName.vi
- [x] Source/APIs/Stimulus/ControlValue/Controls/Listbox/SetValueByItemIndex.vi
- [x] Source/APIs/Stimulus/ControlValue/Controls/Listbox/SetValueByItemName.vi


Stop searching the for loop, if any one of the item was disabled from the given index, in these VIs

- [x] Source/APIs/Stimulus/ControlValue/Controls/Listbox/SetValueByArrayIndex.vi
- [x] Source/APIs/Stimulus/ControlValue/Controls/Listbox/SetValueByArrayName.vi

File: Source/Test/Tests/ListBoxTests/Stimulus/LB_NameArray_Tests.vi
Comment (Giri-anandh) [reply Source/Test/Tests/ListBoxTests/Stimulus/LB_NameArray_Tests.vi:1]: Removed the breakpoint

File: Source/APIs/Stimulus/ControlValue/Controls/Listbox/SetValueByArrayName.vi
Comment (kavin1730) [inline Source/APIs/Stimulus/ControlValue/Controls/Listbox/SetValueByArrayName.vi:1]: - [ ] Shuffle the array and indexed value in for loop (i.e. search index value from disabled item index)

image

**Review by kavin1730 — CHANGES_REQUESTED** [review]
Update the change mentioned in "Source/APIs/Stimulus/ControlValue/Controls/Listbox/SetValueByArrayName.vi"

File: Source/APIs/Stimulus/ControlValue/Controls/Listbox/SetValueByArrayName.vi
Comment (Giri-anandh) [reply Source/APIs/Stimulus/ControlValue/Controls/Listbox/SetValueByArrayName.vi:1]: swapped the inputs to the for loop

**Review by kavin1730 — APPROVED** [review]
All good.

**Pull request merged**